lms/djangoapps/teams/models.py

```diff
@@ -48,9 +49,11 @@ def create(cls, name, course_id, description, topic_id=None, country=None, langu
         """
 
         team_id = generate_unique_readable_id(name, cls.objects.all(), 'team_id')
+        discussion_id = 'team_id:' + team_id
```

Comment:
seems like a cleaner alternative might be a hash based on the team id, or if possible, uuid.